### PR TITLE
Some minor fixes in Kafka-IO and Client-Cli

### PR DESCRIPTION
**Pull request by murong00**
Motivation

Kafka IO/Cli doc minor improvement and small flaws in KafkaSinkConfig and CmdProduce.

Modifications

Amend Kafka IO/Cli doc and modify defaultValue of maxRequestSize in KafkaSinkConfig.

### CODE REVIEW

**Comment by jerrypeng** [comment]
@murong00 thanks for the fixes.  Can you please fix the merge conflicts in this PR.

File: pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSinkConfig.java [hunk 1]
```diff
@@ -60,7 +60,7 @@
             "The batch size that Kafka producer will attempt to batch records together before sending them to brokers.")
     private long batchSize = 16384L;
     @FieldDoc(
-        defaultValue = "16384",
+        defaultValue = "1048576",
```
Comment (ambition119) [inline pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSinkConfig.java]: long type value is 1048576L, so defaultValue is 1048576l or 1048576L.

Comment (ambition119) [reply pulsar-io/kafka/src/main/java/org/apache/pulsar/io/kafka/KafkaSinkConfig.java]: I just [PR](https://github.com/apache/pulsar/blob/d0f8aa6c7d06ad3ad22b373c390fd7b1db61d434/pulsar-io/nifi/src/main/java/org/apache/pulsar/io/nifi/NiFiConfig.java#L71) . if value int value, I also fix this.

**Comment by sijie** [comment]
run java8 tests

**Pull request merged**